firefox-ios/Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -2721,6 +2703,34 @@ class BrowserViewController: UIViewController,
         }
     }
 
+    private func handleAddressField(type: FormAutofillPayloadType?,
```

Comment:
Could we consider a more specific name for this function? Maybe `handleFoundAddressFieldValue`? `The `BrowserViewController` is... large, so the more specific the better!

Reply:
Sure. That makes sense. Thank you for your suggestion! I've applied the change and rebased the PR.